src/misc/ErrorHandlerImpl.js

```diff
@@ -394,6 +393,18 @@ export function showNotAvailableForFreeDialog(isInPremiumIncluded: boolean) {
 	}
 }
 
+export function showMoreStorageNeededOrderDialog(messageIdOrMessageFunction: TranslationKey | lazy<string>): Promise<void> {
```

Comment:
Can you move this out of ErrorHandler (mb SubscriptionUtils) and rebase please?

Reply:
sure